packages/enricher/src/definitions.ts

```diff
@@ -1,14 +1,62 @@
-export const ontologyUrl = 'https://colonialcollections.nl/schema#'; // Internal ontology
+import {z} from 'zod';
 
-export interface BasicEnrichment {
-  id: string;
-}
+export const ontologyUrl =
+  'https://data.colonialcollections.nl/schemas/nanopub#'; // Internal ontology
+
+// We currently have just one version of our ontology
+export const ontologyVersionIdentifier = 'Version1';
 
-export interface Enrichment extends BasicEnrichment {
-  about: string;
-  description: string;
-  source: string;
-  creator: string;
-  license: string;
-  dateCreated: Date;
+// An enrichment can be about these types
+export enum AdditionalType {
+  Creator = 'creator',
+  DateCreated = 'dateCreated',
+  Description = 'description',
+  Inscription = 'inscription',
+  Material = 'material',
+  Name = 'name',
+  Subject = 'subject',
+  Technique = 'technique',
+  Types = 'types',
```

Comment:
Why is 'Types' plural but all other fields singular?

Reply:
Wel-spotted! Fixed.